src/components/layout/Layout.js

```diff
@@ -0,0 +1,46 @@
+import { Link, Outlet, useLocation } from "react-router-dom";
+
+import { useMediaQuery } from "../../hooks/useMediaQuery.js";
+import logo from "../../assets/logo_w153x3.png";
+import logoText from "../../assets/logo_w81x3.png";
+import profileImage from "../../assets/profile_image_w40x3.png";
+import "./Layout.css";
+
+export default function Layout() {
+  const { pathname } = useLocation();
+  const media = useMediaQuery();
+
+  const getLogo = (media) => {
+    return media !== "MOBLE" ? logo : logoText;
+  };
+
+  const checkPath = (path) => {
+    return pathname === path ? "header__navItem--active" : undefined;
+  };
+
+  return (
+    <>
+      <header className="header">
+        <Link to="/">
+          <img
+            className="header__logo"
+            src={getLogo(media)}
+            alt="판다마켓 바로가기"
+          />
+        </Link>
+        <nav className="header__nav">
+          <ul className="header__navList">
+            <li className={checkPath("/boards")}>
+              <Link to="/boards">자유게시판</Link>
+            </li>
```

Comment:
P1:
react-router-dom 라이브러리의 NavLink component를 사용하시면 더 간단하게 구현이 가능합니다.

```suggestion
            <li className={checkPath("/boards")}>
              <NavLink to="/boards" className={({ isActive }) => isActive ? "header__navItem--active" : ""}>자유게시판</Link>
            </li>
```

*react-router-dom 문서
https://reactrouter.com/en/main/components/nav-link